.eslintrc.js

```diff
@@ -31,6 +31,26 @@ module.exports = {
         'promise/prefer-await-to-callbacks': 'off',
       },
     },
+    {
```

Comment:
Shouldn't these be implemented in `typescript.js` where all other typescript eslint config is placed?

Reply:
This requires the tsconfig.json so this is required. We can look in a later stage if we want to promote some of these settings to the plugin config.